scripts/dev-template.mjs

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env zx
+
+import fs from 'fs/promises'
+import path from 'path'
+
+const template = process.argv[3]
```

Comment:
```suggestion
const template = process.argv[3]
const templatePath = `packages/cli/templates/${template}`
const newConfigPath = `${templatePath}/node_modules/.${template}-vite.config.ts`
await $`cp ${templatePath}/vite.config.ts ${newConfigPath}`
```

Reply:
You can select multi-lines and suggest :)

https://github.com/storyblok/field-plugin/assets/499898/7ebfea6f-4067-491c-9571-c96e43b2c993

anyway, thanks I'll apply this.

Reply:
Really cool @eunjae-lee. 

Thanks for sharing, it'll be helpful.

Reply:
`const template = process.argv[3]`

Since the `template` parameter is mandatory for the script to work properly, I'm wondering if it would be nice to check if before and if it isn't provided, print a better error message.

What do you think @eunjae-lee ?

Reply:
This script is run by `bump-version`, and we don't run it directly. So this script is supposed to get the argument all the time. However, I just added it for better read :) 

https://github.com/storyblok/field-plugin/pull/264/commits/f7726dca5a2efbddf6431c8903078665c10c5be7